packages/shared/src/urbit/channel.ts

```diff
@@ -172,14 +172,23 @@ export type KindDataChat = {
   chat: null | { notice: null };
 };
 
-export type KindData = KindDataDiary | KindDataChat | KindDataHeap;
-export type Kind = 'heap' | 'diary' | 'chat';
+export type KindDataCustom = {
+  custom: string;
```

Comment:
nit: in an extreme of the custom channels world, everything will end up with kind `custom` – aesthetically, I wonder if there's another term that we could use to support that. ("user"?)

Reply:
I think at that point we'd just change our API so that kind-data goes away and we just have a post-type field or w/e.